app/plugin_install.go

```diff
@@ -107,6 +123,27 @@ func (a *App) RemovePlugin(id string) *model.AppError {
 }
 
 func (a *App) removePlugin(id string) *model.AppError {
+	if rmErr := a.removePluginLocally(id); rmErr != nil {
+		return rmErr
+	}
+
+	// Remove bundle from the file store.
+	storePluginFileName := filepath.Join("./plugins", id) + ".tar.gz"
+	bundleExist, fileErr := a.FileExists(storePluginFileName)
+	if fileErr != nil {
+		return model.NewAppError("removePlugin", "app.plugin.remove_bundle.app_error", nil, fileErr.Error(), http.StatusInternalServerError)
+	}
+
+	if bundleExist {
```

Comment:
invert and return early?

Reply:
Fixed. 
However, I feel that when you return early with no error and have a medium piece of code afterwards is not obvious that is dependent on that condition. In the future, someone may be inclined to add something new to the end of the method and not notice that it can return early based on an unrelated condition. Just something to keep an eye out.